sass/_main.scss

```diff
@@ -552,3 +552,15 @@ hr {
   overflow: hidden;
   z-index:-99;
 }
+
+.ul-black {
+  border-bottom: 1px solid black;
+}
+
+.ul-blue {
+  border-bottom: 1px solid #4330fc;
+}
+
+.ul-green {
```

Comment:
These are redundant. `.b—black`, `.b--blue2` and `.b--green2` are in the class namespace and come from indigo.

Reply:
Good catch, resolved now.